services/appflowy-collaborate/src/group/group_init.rs

```diff
@@ -1034,7 +1030,7 @@ impl CollabPersister {
       collab.transact_mut().apply_update(update).map_err(|err| {
```

Comment:
Would it be better to acquire a transaction once and reuse it, rather than obtaining a new one for each update?